projects/sonar/src/app/deposit/editor/editor.component.html

```diff
@@ -24,142 +24,174 @@
-                <span class="caret"></span>
-              </button>
-              <ul id="dropdown-import" *dropdownMenu class="dropdown-menu" role="menu" aria-labelledby="button-basic">
-                <li role="menuitem">
-                  <a class="dropdown-item" href
-                    (click)="$event.preventDefault(); extractPdfMetadata()">{{ 'Analyze uploaded PDF' | translate }}</a>
-                </li>
-              </ul>
-            </div>
-          </div>
-          <div class="col-6 text-right">
-            <button type="submit" class="btn btn-primary">
-              <i class="fa fa-floppy-o mr-2"></i>
-              {{ 'Save' | translate }}
-            </button>
-            <button type="button" class="btn btn-primary ml-2" (click)="publish()" *ngIf="isFinished()">
-              <i class="fa fa-check mr-2"></i>
-              <span *ngIf="isAdminUser; else submitLabel">
-                {{ 'Publish' | translate }}
-              </span>
-              <ng-template #submitLabel>{{ 'Submit' | translate }}</ng-template>
-            </button>
-          </div>
-        </div>
-      </div>
-      <div class="col-sm-4 text-center text-sm-right">
-        <div class="btn-group" role="group" aria-label="Basic example">
-          <button type="button" class="btn" [ngClass]="{
-              'btn-primary': view === viewItem.key,
-              'btn-light': view !== viewItem.key
-            }" *ngFor="
-              let viewItem of [
-                { key: 'form', icon: 'tasks' },
-                { key: 'preview', icon: 'desktop' },
-                { key: 'json', icon: 'code' }
-              ]
-            " (click)="view = viewItem.key">
-            <i [class]="'fa fa-' + viewItem.icon"></i>
-          </button>
-        </div>
-      </div>
-    </div>
     <div class="row">
       <div class="col-sm-8">
-        <ng-container *ngIf="view === 'form'">
-          <formly-form [form]="form" [model]="model" [fields]="fields"></formly-form>
-          <div class="form-group row" *ngIf="currentStep === 'metadata'">
-            <label class="col-sm-4">{{ 'ID' | translate }}</label>
-            <div class="col-sm-8"><i class="fa fa-lock mr-2"></i> {{ deposit.pid }}</div>
-          </div>
-          <ng-container *ngIf="currentStep === 'diffusion'">
-            <div class="form-group row">
-              <label class="col-sm-4">{{ 'Created date' | translate }}</label>
-              <div class="col-sm-8">
-                <i class="fa fa-lock mr-2"></i> {{ createdAt | dateTranslate: 'medium' }}
+        <ng-container *ngIf="view === 'form' && currentStep !== 'diffusion'">
+          <div class="row mb-4">
+            <div class="col-6">
+              <div class="btn-group" dropdown *ngIf="currentStep === 'metadata' && mainFile">
+                <button id="import-button" dropdownToggle type="button" class="btn btn-primary dropdown-toggle"
```

Comment:
There is a select menu in Bootstrap 4 if applicable: https://getbootstrap.com/docs/4.0/components/forms/#select-menu

Reply:
I don't think so, we want a dropdown here.